### ISSUE TITLE
How to reduce bundle size?

### ISSUE THREAD

**Issue by maximelebreton** [Turfjs/turf#1846]
Hello there,
this is my `webpack bundle analyzer` report, and as you see, the turf size is pretty big (556ko)!

![image](https://user-images.githubusercontent.com/1072425/76262689-583b8e00-625d-11ea-85db-b9280af33709.png)

I've verified twice, and all my files contains only partial imports, like this:
```js
import {
  getCoords,
  nearestPoint,
  along,
  point
} from "@turf/turf";
```

So I don't understand why this is so big?

Am I doing something wrong?

Thanks!

**Comment by rowanwins** [comment]
Can you confirm which modules you're importing @maximelebreton - there are some which will result in large bundles although the ones above shouldn't be particuarly problematic.

And which version of turfjs do you have in your package-lock.json?

**Comment by mfedderly** [comment]
For now the @turf/turf dependency isn't set up for proper tree shaking (I'd like to fix that). In the meantime, I've had success shrinking my total bundle size by using the sub packages `@turf/nearest-point` and similar.

**Comment by 3zzy** [comment]
@mfedderly Is this still true in Aug 2021 or it has been addressed? Thanks.

**Comment by mfedderly** [comment]
It should have been addressed on the 6.2.0 and later builds. @turf/turf should be smaller than it was, if you can use a @turf/* single module that would certainly be the most likely to give you a small bundle size.
